packages/github/__tests__/lib.test.ts

```diff
@@ -29,18 +41,13 @@ describe('@actions/context', () => {
 
   it('returns attributes from the repository payload', () => {
     delete process.env.GITHUB_REPOSITORY
-
-    context.payload.repository = {
-      name: 'test',
-      owner: {login: 'user'}
-    }
     expect(context.repo).toEqual({owner: 'user', repo: 'test'})
   })
 
   it("return error for context.repo when repository doesn't exist", () => {
     delete process.env.GITHUB_REPOSITORY
-
-    context.payload.repository = undefined
+    delete process.env.GITHUB_EVENT_PATH
```

Comment:
should an after all restore the original values? otherwise may affect test suites that execute after

Reply:
The beforeeach function sets up these values
https://github.com/actions/toolkit/pull/292/files#diff-75df1656ab4669c70d84cd99613c8a6eL10

Reply:
should an after-all cleanup? so the changes don't spill-over and affect other test suites?

Reply:
Oh sorry, I misunderstood your previous comment. Yeah, I'll go ahead and update this